src/components/utdragFraSykefravaeret/UtdragFraSykefravaeret.tsx

```diff
@@ -208,31 +209,28 @@ const SamtalereferatWrapper = styled.div`
   margin-bottom: 2em;
 `;
 
-interface SamtalereferatProps {
-  fnr: string;
-}
-
-export const Samtalereferat = ({ fnr }: SamtalereferatProps) => (
-  <SamtalereferatWrapper>
-    <Undertittel tag={"h3"}>{tekster.samtalereferat.header}</Undertittel>
-    <Lenke
-      className="lenke"
-      href={`https://modapp${finnMiljoStreng()}.adeo.no/modiabrukerdialog/person/${fnr}#!meldinger`}
-      target="_blank"
-    >
-      {tekster.samtalereferat.lenkeTekst}
-    </Lenke>
-  </SamtalereferatWrapper>
-);
+export const Samtalereferat = () => {
+  const fnr = useValgtPersonident();
+  return (
+    <SamtalereferatWrapper>
+      <Undertittel tag={"h3"}>{tekster.samtalereferat.header}</Undertittel>
+      <Lenke
+        className="lenke"
+        href={`https://modapp${finnMiljoStreng()}.adeo.no/modiabrukerdialog/person/${fnr}#!meldinger`}
+        target="_blank"
+      >
+        {tekster.samtalereferat.lenkeTekst}
+      </Lenke>
+    </SamtalereferatWrapper>
+  );
+};
 
 interface UtdragFraSykefravaeretProps {
   aktivePlaner: OppfolgingsplanDTO[];
-  fnr: string;
```

Comment:
Du trenger ikke gjøre det nå, men vi kunne vel kanskje hentet `aktivePlaner` inne i denne komponenten i stedet for å sende det som prop? 🤔

Reply:
Ja, kan nok forenkle det mer - fikser egen PR 👍